src/activities/ActivityUsageEntity.js

```diff
@@ -264,4 +264,126 @@ export class ActivityUsageEntity extends Entity {
 	isPublished() {
 		return this._entity && this._entity.hasClass(Classes.activities.draftPublishedEntity) && this._entity.hasClass(Classes.activities.published);
 	}
+
+	/**
+	 * @returns {string} Score out of value of the activity usage
+	 */
+	scoreOutOf() {
+		if (this.canEditScoreOutOf()) {
```

Comment:
This is getting the value from the action if it exists (i.e. if user can edit), and from the properties otherwise. Perhaps this is confusing, I can change it to explicitly check if the action exists.

Reply:
Is there a need to fetch it from the action though? Wouldn't that always be the same as the property anyway, or are there situations where they can differ?

Reply:
Since the action fields are supposed to represent the current state of the form element, it is more "proper" to get the values from there. However if the user has no permissions then there is no action, hence the fallback to properties. This is how we ended up doing it in Activity Feed, to try to keep things as conforming to specs as possible.

Reply:
True, mostly was just thinking you'll either get the same value as the property anyway, or you'll end up falling back to the property value anyway, so didn't really see the need for the extra complexity. But I also don't see there being any harm to doing it one way or the other, so long as its done consistently!